pkg/odo/cli/component/list.go

```diff
@@ -104,57 +144,120 @@ func (lo *ListOptions) Run() (err error) {
 		}
 		return nil
 	}
-	var components component.ComponentList
 
-	if lo.allAppsFlag {
-		// retrieve list of application
-		apps, err := application.List(lo.Client)
+	// non --path workflow below
+	// read the code like
+	// -> experimental
+	//	or	|-> --all-apps
+	//		|-> the current app
+	// -> non-experimental
+	//	or	|-> --all-apps
+	//		|-> the current app
+
+	// experimental workflow
+
+	if experimental.IsExperimentalModeEnabled() {
+
+		var dpl *appsv1.DeploymentList
+		var err error
+
+		// TODO: wrap this into a component list for docker support
```

Comment:
left a TODO here by mistake?

Reply:
docker support marker for someone who would implement it